examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/doc/google/cloud/speech/v1/cloud_speech.rb

```diff
@@ -0,0 +1,482 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module Google
+  module Cloud
+    module Speech
+      module V1
+        # The top-level message sent by the client for the `Recognize` method.
+        # @!attribute [rw] config
+        #   @return [Google::Cloud::Speech::V1::RecognitionConfig]
+        #     *Required* Provides information to the recognizer that specifies how to
+        #     process the request.
+        # @!attribute [rw] audio
+        #   @return [Google::Cloud::Speech::V1::RecognitionAudio]
+        #     *Required* The audio data to be recognized.
+        class RecognizeRequest; end
```

Comment:
Noting that if we go the proto-plus route, these would be the proto definitions.

Reply:
What is "the proto-plus route"?

Reply:
It's probably easier to discuss this in person, but it's related to the doc that we chatted about listing any/all existing pain points using proto message types. We need that to define "proto plus".